src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -648,6 +679,12 @@ public OpenIdConnectHandler(HttpClient backchannel)
             ticket = userInformationReceivedContext.AuthenticationTicket;
             user = userInformationReceivedContext.User;
 
+            Options.ProtocolValidator.ValidateUserInfoResponse(new OpenIdConnectProtocolValidationContext()
+            {
+                ClientId = Options.ClientId,
+                ProtocolMessage = message,
+            });
+
             var identity = (ClaimsIdentity)ticket.Principal.Identity;
             var subjectClaimType = identity.FindFirst(ClaimTypes.NameIdentifier);
```

Comment:
This is a potential problem since there could be some mapping going on here. If users mapped 'sub' to something else in the ClaimsIdentity in the ticket. We should open a RC tracking issue.